Makefile

```diff
@@ -13,6 +13,23 @@ export_full = $(eval export $(if $(call err_if_empty,$(1)),$(1)))
 # Evaluate to the value of $(1) if $(CI) is the literal string "true", else $(2)
 if_ci_else = $(if $(findstring true,$(CI)),$(1),$(2))
 
+##### Important image release and source details #####
+
+export CENTOS_STREAM_RELEASE = 8
+
+export FEDORA_RELEASE = 37
+export FEDORA_IMAGE_URL = https://dl.fedoraproject.org/pub/fedora/linux/development/37/Cloud/x86_64/images/Fedora-Cloud-Base-37-20220830.n.0.x86_64.qcow2
+export FEDORA_CSUM_URL = https://dl.fedoraproject.org/pub/fedora/linux/development/37/Cloud/x86_64/images/Fedora-Cloud-37-x86_64-20220830.n.0-CHECKSUM
+export FEDORA_ARM64_IMAGE_URL = https://dl.fedoraproject.org/pub/fedora/linux/development/37/Cloud/aarch64/images/Fedora-Cloud-Base-37-20220830.n.0.aarch64.qcow2
+export FEDORA_ARM64_CSUM_URL = https://dl.fedoraproject.org/pub/fedora/linux/development/37/Cloud/aarch64/images/Fedora-Cloud-37-aarch64-20220830.n.0-CHECKSUM
```

Comment:
no biggie, but may be convenient to have these URLs crafted using something like: $(BASE_URL)/$(FEDORA_RELEASE)/$(FOO)/$(BAR)

Reply:
It's a good suggestion. But would make copy-pasting the link harder.    Actually I was thinking about if/how they could be looked up based on the release number.  Especially the development case, the filenames change daily, sometimes multiple times per day.  The release filenames less so, but I have seen them change too.

Reply:
https://github.com/containers/automation_images/issues/190